packages/neuron-ui/src/locales/en.json

```diff
@@ -335,6 +336,7 @@
       "full-addr-info": "Additional software or dApp interface might be required to operate the assets on this address. Please double check its validity.",
       "decimal-range": "Decimal should be an interger of {{range}}(inclusive)",
       "experimental-message": "This is an experimental feature, it could change at any time. Please use with caution.",
+      "rebuild-sync": "To achieve better user experience, Neuron has adopted a brand new index mechanism, which demands a rebuilding of cache database (estimated 30 ~ 60min).\nSorry for the inconvenience.",
```

Comment:
Wording suggestion:
"a brand new index mechanism" => "a brand new indexing mechanism"

Reply:
"To achieve better user experience" => "For better user experience"

Reply:
"which demands" => "which requires"

Reply:
This kind of update should be confirmed in the PRD issue..

Reply:
That is why we need PR review, right? 😛 
They are just some suggestions. The original version might be better.

Reply:
> That is why we need PR review, right? 😛
> They are just some suggestions. The original version might be better.

😣 Actually I don't make a judgement on the expression, I just want to make a consensus among us. Otherwise, the content of notification should fail in tests.